client/src/components/CredentialsInputBox/CredentialsInputBox.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import "./CredentialsInputBox.css";
+
+export default function CredentialsInputBox() {
+  return (
+    <div className="cred-layout">
+      <b className="cred-input-header">Email</b>
```

Comment:
If I'm not mistaken, I don't think `<b>` should be included in here since the text doesn't seem to be bolded to me.
A `<p>` tag or even a regular `<div>` might suffice.

Reply:
Fixed: changed to `<p>` tags. Initially, this had some styling problems (`<p>` has different default margins) but that has been resolved.